docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -292,6 +304,43 @@ type LocalObjectTemplate struct {
   // offered by a provider.
   Ref *corev1.ObjectReference `json:"ref"`
 }
+
+// MachineHealthCheckClass defines a MachineHealthCheck for a group of Machines.
+type MachineHealthCheckClass struct {
+  // UnhealthyConditions contains a list of the conditions that determine
+  // whether a node is considered unhealthy. The conditions are combined in a
+  // logical OR, i.e. if any of the conditions is met, the node is unhealthy.
```

Comment:
Should we add a note about this field being defaulted when the actual MHC will be created?

Reply:
Added a note on defaulting above the struct.